testsuite/MDAnalysisTests/coordinates/base.py

```diff
@@ -714,7 +714,12 @@ def _from_coords(self, p, v, f):
 
         return ts
 
-    def _check_from_coordinates(self, p, v, f):
+    @pytest.mark.parametrize('p, v, f',
```

Comment:
if we keep reusing the pattern of `itertools.product()` it's probably better to define it as a generator outside the class

Reply:
I changed this to a fixture `some_ts` in the other occurrences. Here this isn't possible.